apps/web/components/ui/UsernameAvailability/index.tsx

```diff
@@ -66,6 +65,7 @@ export const UsernameAvailabilityField = ({
             onSuccessMutation={onSuccessMutation}
             onErrorMutation={onErrorMutation}
             addOnLeading={usernamePrefix}
+            disabled={!!user.organization}
```

Comment:
We don't want org members to be able to change their username so the user@org.com continues to correspond to org.cal.com/user